eos/b-decays/b-to-d-l-nu.cc

```diff
@@ -54,6 +61,9 @@ namespace eos
             model(Model::make(o.get("model", "SM"), p, o)),
             parameters(p),
             m_B(p["mass::B_" + o.get("q", "d")], u),
+            m_b_MSbar(p["mass::b(MSbar)"], u),
+            m_d_MSbar(p["mass::d(MSbar)"], u),
```

Comment:
1. We use the down quark mass by default, but can (and do) deviate from this by setting the Option e.g. ```q=u```.
Moreover, the u and d quark masses are negligible compared to the b quark mass. I would suggest to drop the light quark masses completely.

2. ```mass::b(MSbar)``` is an internal parameter and should not be used here. Use ```model->m_b_msbar(mu)``` instead please.